src/fasttext_lite/core.py

```diff
@@ -217,3 +223,108 @@ def _get_original_label(self, adjusted_label: str):
 
     def _get_original_label_index(self, adjusted_label: str):
         return self.original_labels.index(self._get_original_label(adjusted_label))
+
+
+class FastTextMultiOutputClassifier(FastTextClassifier):
+    def __init__(
+        self,
+        lr: float = 0.1,
+        dim: int = 100,
+        ws: int = 5,
+        epoch: int = 5,
+        minCount: int = 1,
+        minCountLabel: int = 1,
+        minn: int = 0,
+        maxn: int = 0,
+        neg: int = 5,
+        wordNgrams: int = 1,
+        bucket: int = 2000000,
+        lrUpdateRate: int = 100,
+        t: float = 0.0001,
+        label: str = "__label__",
+        verbose: int = 2,
+        thread: int = 2,
+    ):
+        super().__init__(
+            lr=lr,
+            dim=dim,
+            ws=ws,
+            epoch=epoch,
+            minCount=minCount,
+            minCountLabel=minCountLabel,
+            minn=minn,
+            maxn=maxn,
+            neg=neg,
+            wordNgrams=wordNgrams,
+            loss="ova",
+            bucket=bucket,
+            lrUpdateRate=lrUpdateRate,
+            t=t,
+            label=label,
+            verbose=verbose,
+            thread=thread,
+        )
```

Comment:
This is very un-DRY, but sklearn requires all parameters to be explicit in the `__init__` method's signature for estimators. This is to ensure consistent behavior with various functions that rely on explicit attribute setting, such as `get_params`. So using `**kwargs` throws a runtime error. Still, maybe there is a better way to do this.